### ISSUE TITLE
What to do after generating the lib

### ISSUE THREAD

**Issue by blu3beri** [microsoft/react-native-tscodegen#47]
Hi!

I have been messing around a bit with this library and everything seems to be working fine but I am not too sure on how to continue.

I have created a project with [bob](https://github.com/callstack/react-native-builder-bob), to create a cpp turbomodule library for RN, and generated the `lib` via the provided example config. Everything works and the cpp code seems to be generated correctly.

However, since the demo application is out yet, I do not see where I would write my cpp implementation of the generated function. It does not help that my cpp is quite lacking, but I just could not figure it out.

tl;dr:

1. How do I use the generated cpp code to add my implementation
2. Is there any ETA on the demo library where tscodegen is used?

cheers!
